### PR TITLE
Sm 1053/tracking bug sm version

### PR DESCRIPTION
**Pull request by MarcMcIntosh**
## Context
https://linear.app/prismic/issue/SM-1053/[tracking-bug]-aapm-i-would-like-to-know-which-version-of-sm-users-are

## Checklist before requesting a review
- [ ] I hereby declare my code ready for review.
- [ ] If it is a critical feature, I have added tests.
- [ ] The CI is successful.
- [ ] If there could backward compatibility issues, it has been discussed and planned.

### CODE REVIEW

File: cypress/helpers/repository.js [hunk 1]
```diff
@@ -24,7 +27,7 @@ export function pushLocalChanges(numberOfChanges) {
     // The time to wait depends on the number of changes
     menu
       .changesNumber({
-        timeout: 5000 * (numberOfChanges + 1),
+        timeout: 2 * 60 * 1000 * (numberOfChanges + 1),
```
Comment (mathieud) [inline cypress/helpers/repository.js:30]: Do we still need this with the new transactional push?

File: cypress/helpers/repository.js [hunk 2]
```diff
@@ -11,7 +11,10 @@ export function pushLocalChanges(numberOfChanges) {
 
   if (numberOfChanges !== undefined) {
     // checking number of changes
-    menu.changesNumber().contains(numberOfChanges).should("be.visible");
+    menu
+      .changesNumber()
+      // .contains(numberOfChanges) <-- see this... it doesn't work 🙀
```
Comment (mathieud) [inline cypress/helpers/repository.js:16]: Worked (or appeared to) until now. What triggered this?

File: cypress/pages/BaseBuilder.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 export class BaseBuilder {
   get saveButton() {
-    return cy.contains("Save to File System");
+    return cy.contains("Save to File System", { timeout: 60 * 1000 });
```
Comment (mathieud) [inline cypress/pages/BaseBuilder.js]: Is the page very slow to load?

File: packages/slice-machine/components/AppLayout/Navigation/Menu/Navigation/VideoItem.tsx [hunk 4]
```diff
@@ -25,11 +23,7 @@ const VideoItem: FC<VideoItemProps> = ({
   const videoUrl = VIDEO_YOUTUBE_PLAYLIST_LINK;
 
   const handleClose = () => {
-    void Tracker.get().trackClickOnVideoTutorials(
-      framework,
-      sliceMachineVersion,
```
Comment (mathieud) [inline packages/slice-machine/components/AppLayout/Navigation/Menu/Navigation/VideoItem.tsx:30]: Is Come ok with this change?

File: packages/slice-machine/server/src/api/tracking.ts [hunk 5]
```diff
@@ -16,11 +17,18 @@ export function sendEvents(
   userId?: string,
   intercomHash?: string
 ): void {
+  const pathToPackage = path.resolve(__dirname, "..", "..", "..");
+  const pkg = retrieveJsonPackage(pathToPackage);
+  const version = pkg.content?.version ?? "";
```
Comment (mathieud) [inline packages/slice-machine/server/src/api/tracking.ts]: Do we need to do this on every event? Why not just once when the module loads?

File: cypress/helpers/repository.js (hunk 2 above)
Comment (MarcMcIntosh) [reply cypress/helpers/repository.js:16]: Retries, when it fails and retires the number of changes was 2

File: packages/slice-machine/components/AppLayout/Navigation/Menu/Navigation/VideoItem.tsx (hunk 4 above)
Comment (MarcMcIntosh) [reply packages/slice-machine/components/AppLayout/Navigation/Menu/Navigation/VideoItem.tsx:30]: ...wait looks like I deleted to much there :)

Comment (MarcMcIntosh) [reply packages/slice-machine/components/AppLayout/Navigation/Menu/Navigation/VideoItem.tsx:30]: no this is correct, the version is added on the `/s` route

File: packages/slice-machine/server/src/api/tracking.ts (hunk 5 above)
Comment (MarcMcIntosh) [reply packages/slice-machine/server/src/api/tracking.ts]: Can do :)

File: cypress/pages/BaseBuilder.js (hunk 3 above)
Comment (MarcMcIntosh) [reply cypress/pages/BaseBuilder.js]: It did on my machine, I can try and remove it and see what happens :)

File: cypress/helpers/repository.js (hunk 1 above)
Comment (MarcMcIntosh) [reply cypress/helpers/repository.js:30]: I think so, depends on how the screenshots are handled

File: cypress/pages/BaseBuilder.js (hunk 3 above)
Comment (MarcMcIntosh) [reply cypress/pages/BaseBuilder.js]: https://github.com/prismicio/slice-machine/actions/runs/4292541292

Comment (MarcMcIntosh) [reply cypress/pages/BaseBuilder.js]: Seems just have been on my machine :)

File: cypress/helpers/repository.js (hunk 1 above)
Comment (MarcMcIntosh) [reply cypress/helpers/repository.js:30]: Looks like it: https://github.com/prismicio/slice-machine/actions/runs/4287746526/jobs/7468979692#step:8:819
pushing screen shots takes a while longer

**Comment by MarcMcIntosh** [comment]
updated the branch :/

**Pull request closed without merging**